src/main/java/wooteco/subway/config/WebMvcConfig.java

```diff
@@ -1,37 +1,28 @@
 package wooteco.subway.config;
 
+import java.util.List;
+
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
 import wooteco.subway.web.member.LoginMemberMethodArgumentResolver;
-import wooteco.subway.web.member.interceptor.BasicAuthInterceptor;
 import wooteco.subway.web.member.interceptor.BearerAuthInterceptor;
-import wooteco.subway.web.member.interceptor.SessionInterceptor;
-
-import java.util.List;
 
 @Configuration
 public class WebMvcConfig implements WebMvcConfigurer {
-    private final BasicAuthInterceptor basicAuthInterceptor;
-    private final SessionInterceptor sessionInterceptor;
     private final BearerAuthInterceptor bearerAuthInterceptor;
     private final LoginMemberMethodArgumentResolver loginMemberArgumentResolver;
 
-    public WebMvcConfig(BasicAuthInterceptor basicAuthInterceptor,
-                        SessionInterceptor sessionInterceptor,
-                        BearerAuthInterceptor bearerAuthInterceptor,
-                        LoginMemberMethodArgumentResolver loginMemberArgumentResolver) {
-        this.basicAuthInterceptor = basicAuthInterceptor;
-        this.sessionInterceptor = sessionInterceptor;
+    public WebMvcConfig(BearerAuthInterceptor bearerAuthInterceptor,
+        LoginMemberMethodArgumentResolver loginMemberArgumentResolver) {
         this.bearerAuthInterceptor = bearerAuthInterceptor;
         this.loginMemberArgumentResolver = loginMemberArgumentResolver;
     }
 
     @Override
     public void addInterceptors(InterceptorRegistry registry) {
-        registry.addInterceptor(basicAuthInterceptor).addPathPatterns("/me/basic");
-        registry.addInterceptor(sessionInterceptor).addPathPatterns("/me/session");
-        registry.addInterceptor(bearerAuthInterceptor).addPathPatterns("/me/bearer");
+        registry.addInterceptor(bearerAuthInterceptor).addPathPatterns("/me/bearer").addPathPatterns("/favorites");
```

Comment:
보통 인증처리 관련 인터셉터를 걸땐 전체를 걸고 인증처리 하지 않을 api만 exclude 해주는 것이 인증 누락될 확률이 낮을 것 같아요 :) 
```suggestion
        registry.addInterceptor(bearerAuthInterceptor)
                .addPathPatterns("/me/bearer")
                .addPathPatterns("/favorites");
```

Reply:
네! 수정하였습니다. 감사합니다. 
추가로 동일한 /members path라도 post,get,put.delete 에 따라 다르게 intercepter를 등록하는 방법이 있을까요?!

Reply:
찾아봤을때는 path로만 등록 가능해보여요 ㅜㅜ 
혹시 늦게라도 찾게되면 말씀드릴게요 :)

Reply:
네! 저도 혹시 알게되면 바로 말씀드리겠습니당! 감사합니다~~